commentTone/src/main/java/com/whz/commenttone/rabbitmq/CommentToneSubscriber.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright © 2020, Fachgruppe Informatik WHZ <help.kb1prb13@gmail.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.whz.commenttone.rabbitmq;
+
+import com.google.gson.GsonBuilder;
+import com.rabbitmq.client.*;
+import com.whz.commenttone.model.CommentTone;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+import java.util.Random;
+import java.util.concurrent.TimeoutException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class can be used to receive a json message from rabbitmq.
+ *
+ * @since 1.0.0
+ */
+public class CommentToneSubscriber {
+
+  private final String serviceName =
+      Optional.ofNullable(System.getenv("RABBITMQ_SERVICE")).orElse("localhost");
+
+  private final String publishExchangeName =
+      Optional.ofNullable(System.getenv("RABBITMQ_PUBLISH_EXCHANGE")).orElse("commentTone");
+  private final String consumeExchangeName =
+      Optional.ofNullable(System.getenv("RABBITMQ_CONSUME_EXCHANGE")).orElse("feedback");
+  private final String exchangeType =
+      Optional.ofNullable(System.getenv("RABBITMQ_EXCHANGE_TYPE")).orElse("fanout");
+
+  private final ConnectionFactory connectionFactory;
+
+  private final CommentTonePublisher<CommentTone> publisher;
+
+  private final Logger logger = LoggerFactory.getLogger(CommentToneSubscriber.class);
+
+  /**
+   * Create a subscriber which is connected to a specific rabbitmq instance.
+   *
+   * @since 1.0.0
+   */
+  public CommentToneSubscriber() {
+    this.connectionFactory = new ConnectionFactory();
+    connectionFactory.setHost(serviceName);
+
+    publisher = new CommentTonePublisher<>(serviceName);
+  }
+
+  /**
+   * Consume a new message from the rabbitmq instance.
+   *
+   * @throws IOException if an error is encountered
+   * @throws TimeoutException if a blocking operation times out
+   * @since 1.0.0
+   */
```

Comment:
This consuming is one-time (service will instantly get stopped after this in docker).